frontend/packages/client/src/pages/CommunityCreate.js

```diff
@@ -65,13 +66,70 @@ export default function CommunityCreate() {
     // create one object from steps data
     const fields = Object.assign({}, ...Object.values(stepsData));
 
-    const proposalData = {
+    const {
+      contractAddress,
+      listAddrAdmins = [],
+      listAddrAuthors = [],
+      strategies = [],
+    } = fields;
+
+    // validate Flow Addresses used
+    const addressesToValidate = {
+      'Contract Address': [contractAddress],
+      'Admin List': listAddrAdmins.map((e) => e.addr),
+      'Author List': listAddrAuthors.map((e) => e.addr),
+      Strategies: strategies,
```

Comment:
Strategies will be added on https://github.com/DapperCollectives/CAST/pull/62, adding them here to update Pr after #62 is merged